<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSerialConsoleWindow</class>
 <widget class="QMainWindow" name="QSerialConsoleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSerialConsoleWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="consoleFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>520</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>S&amp;ettings</string>
    </property>
    <widget class="QMenu" name="menuProtocol">
     <property name="title">
      <string>Pro&amp;tocols</string>
     </property>
     <addaction name="actionNone"/>
     <addaction name="actionASCII"/>
    </widget>
    <addaction name="actionPortSettings"/>
    <addaction name="separator"/>
    <addaction name="menuProtocol"/>
   </widget>
   <addaction name="menu_Settings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPortSettings">
   <property name="text">
    <string>&amp;Port Settings</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;None</string>
   </property>
  </action>
  <action name="actionASCII">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;ASCII</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
